src/clr-angular/data/datagrid/datagrid-column-toggle-button.spec.ts

```diff
@@ -19,22 +19,7 @@ export default function(): void {
         component = new ClrDatagridColumnToggleButton(columnToggleButtons);
       });
 
-      it('knows if the button isOk()', function() {
```

Comment:
I'd like to have a test here that shows that an `ok` button type is inert and doesn't throw an error.

Reply:
Good idea. Will add this test back in and modify.